remotezip.py

```diff
@@ -259,12 +258,26 @@ def _list_files(url, support_suffix_range, filenames):
     with RemoteZip(url, headers={'User-Agent': 'remotezip'}, support_suffix_range=support_suffix_range) as zip:
         if len(filenames) == 0:
             filenames = zip.namelist()
-        data = [('Length', 'DateTime', 'Name')]
+        data = []
         for fname in filenames:
             zinfo = zip.getinfo(fname)
             dt = datetime(*zinfo.date_time)
             data.append((zinfo.file_size, dt.strftime('%Y-%m-%d %H:%M:%S'), zinfo.filename))
-        print(tabulate(data, headers='firstrow'))
+        _printTable(data, ('Length', 'DateTime', 'Name'), '><<')
+
+
+def _printTable(data, header, align):
+    # get max col width & prepare formatting string
+    col_w = [len(col) for col in header]
+    for row in data:
+        col_w = [max(w, len(str(x))) for w, x in zip(col_w, row)]
+    fmt = '  '.join(f'{{:{a}{w}}}' for w, a in zip(col_w, align + '<' * 99))
```

Comment:
f-strings doesn't work in python 2.7. It's probably time to remove compatibility for it, but that would require a major release that is probably outside the scope of this MR.

Reply:
replaced with format()